server_benchmark.py

```diff
@@ -113,12 +114,18 @@ def benchmarkOneQuery(request, repeatLimit=3, pageLimit=3):
     # return sum(times[2:])/len(times[2:])
     return min(times)
 
-if __name__ == '__main__':
+
+def main(args):
```

Comment:
It's a bit confusing if we have ``args`` referring to two different things here. Maybe change this parameter to ``argList`` or something?